### PR TITLE
Copyright update of more files that have changed this year

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

**Comment by levitte** [comment]
I'll do a separate PR for 1.1.0 and 1.0.2

**Comment by levitte** [comment]
It occured to me that we probably missed a bunch of files last year.  The run for this PR was only a check of files changed since the beginning of this year...  I wonder what happens if I check last year's changes...

File: util/mkerr.pl [hunk 1]
```diff
@@ -702,7 +702,7 @@ sub help
     open(OUT, ">$statefile.new")
         || die "Can't write $statefile.new, $!";
     print OUT <<"EOF";
-# Copyright 1999-2017 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 1999-$YEAR The OpenSSL Project Authors. All Rights Reserved.
```
Comment (paulidale) [inline util/mkerr.pl]: Really?

**Review by paulidale — APPROVED** [review]

**Comment by paulidale** [comment]
Reconfirmed.

I'm not seeing `util/mkerr.pl` having been changed.

**Comment by levitte** [comment]
`util/mkerr.pl` was changed in the first incantation of this PR, you're not seeing it now because I reverted that change

**Comment by levitte** [comment]
Thank you

**Comment by levitte** [comment]
Merged. 48e5119a6b83fd5781174d3b524cb8d1dc411649

**Pull request closed without merging**